week-3/reading-list/script.js

```diff
@@ -1,4 +1,17 @@
-// for the tests, do not modify this array of books
+// // for the tests, do not modify this array of books
+
+function readingList(li) {
+  const shuffledBooks = [...li];    //the three dots is spread syntext that copy the shallow array.
+
+  
+  for (let i = shuffledBooks.length - 1; i > 0; i--) {
+    const indexZ = Math.floor(Math.random() * (i + 1));
```

Comment:
Again, don't be afraid to challenge yourself to come up with more descriptive names :) ,especially when it comes to calculations, etc `shuffledBooks` instead of `indexZ`